app/controllers/courses_controller.rb

```diff
@@ -33,7 +33,11 @@ def index
     @self_accredited_courses = @courses_by_accrediting_provider.delete(@provider.provider_name)
   end
 
-  def show; end
+  def show
+    site_statuses = @course.site_statuses
+    site_statuses = site_statuses.select { |ss| ss.status == 'running' } if @course.is_running?
```

Comment:
should this be 
` site_statuses = site_statuses.select { |ss| ss.status == 'running' } if @course.is_running? || @course.is_new_status?`

Reply:
Not sure. @benilovj ?

Reply:
I think you want to do:

```ruby
site_statuses = @course.site_statuses.select { |ss| ss.in?(["running", "new_status"]) }
```

Reply:
You would still need  the `if @course.is_running?` thought right? So

`site_statuses = @course.site_statuses.select { |ss| ss.status.in?(["running", "new_status"]) } if @course.is_running?`